security/s2a/internal/crypter/ciphersuite.go

```diff
@@ -0,0 +1,109 @@
+package crypter
+
+import (
+	"crypto/sha256"
+	"golang.org/x/crypto/sha3"
+	s2a_proto "google.golang.org/grpc/security/s2a/internal/proto"
+	"hash"
+)
+
+// ciphersuite is the interface for a ciphersuite used in the S2A Half
```

Comment:
Could we write a bit more about what `ciphersuite` does rather than how it will be used? For example, we could write: `ciphersuite contains ciphersuite-specific information and utilities.`

Reply:
My example sentence may be circular, but I think there's a way of wordsmithing it that will clearly explain to the reader what the interface does.